openpdf/src/main/java/com/lowagie/text/pdf/PdfContentByte.java

```diff
@@ -538,6 +566,9 @@ public void resetGrayStroke() {
      * @param   green   the intensity of green. A value between 0 and 1
      * @param   blue    the intensity of blue. A value between 0 and 1
      */
+    private void HelperRGB(int red, int green, int blue) {
+        HelperRGB((float)(red & 0xFF) / 0xFF, (float)(green & 0xFF) / 0xFF, (float)(blue & 0xFF) / 0xFF);
```

Comment:
There are still some MagicNumbers here 0xFF can the exchanged with MAX_INT_COLOR_VALUE.